example-workflows/Technique_1_Slack_Workflow_Builder/README.md

```diff
@@ -0,0 +1,81 @@
+# Technique 1: Slack Workflow Builder
+
+> :memo: This technique requires [a Slack paid plan][plans] to use Workflow
+> Builder.
+
+This technique sends data to Slack using a webhook to start a workflow created
+using Slack [Workflow Builder][wfb].
+
+## Setup
+
+The Slack app manifest included with each example showcases steps of a workflow
+in [Workflow Builder][wfb] along with the [webhook trigger][triggers] inputs.
+
+For details on how to setup this technique in GitHub Actions, read the
+[`README.md`][setup].
+
+## Example workflows
+
+1. [**Format generated files**](#format-generated-files): Message outputs from
+   prior steps.
+2. [**Post release announcements**](#post-release-announcements): Share releases
+   to a channel.
+3. [**Update a channel topic**](#update-a-channel-topic): Highlight the current
+   build status.
+
+### Format generated files
+
+Convert build outputs from earlier GitHub Action steps into a Slack message.
+
+This example uses data from a payload file to [send a message][send_message] to
+a hardcoded channel.
+
+**Related files**:
+
+- [`builds.data.json`](./builds.data.json): Payload file being sent.
+- [`builds.gha.yml`](./builds.gha.yml): GitHub Actions workflow.
+- [`builds.manifest.json`](./builds.manifest.json): Slack app manifest.
+- [`builds.trigger.json`](./builds.trigger.json): Slack webhook trigger.
+
+### Post release announcements
+
+Select a channel to post news about the most recent release to.
+
+This example uses [Slack functions][functions] and inline inputs to do the
+following:
+
+1. Open a form to select a channel.
+2. Send a message to the selected channel.
+3. React with an excited emoji.
```

Comment:
```suggestion
3. React with an `:excited:` emoji.
```

maybe? idk maybe this is obnoxious

Reply:
I think it's fun! Though `:excited:` might be misleading. I think we're celebrating with a `:tada:` 🎉

Reply:
oh then maybe be explicit with that?
```suggestion
3. React with a `:tada:` emoji.
```